### ISSUE TITLE
Using /dp plugin produces JSON error

### ISSUE THREAD

**Issue by mikeerickson** [NotePlan/plugins#45]
**Describe the bug**
When invoking `/dp` it produces the following error (before displaying date selection dialog)

**To Reproduce**
1. Using latest plugin
2. Command - J
3. /dp

**Expected behavior**
Normal dialog to select date format, without displaying JSON error

**Screenshots**

![CleanShot 2021-08-11 at 07 19 03](https://user-images.githubusercontent.com/183153/129046427-8ea16f88-1a83-4823-a7ca-728705d39a86.png)

**System**
 - macOS
 - OS: Big Sur 11.5.1
 - NotePlan Version3.0.26 (646)
 - Date Automationnnn v1.0.8

**Comment by weyert** [comment]
@mikeerickson Looks like you have a syntax error in your configuration. You can test it by copy-pasting the configuration block into this site: https://codebeautify.org/json5-validator

This site should return warnings where the problematic line is. If you want you can also post it here on the Discord and we can have a look. Typically it's a missing `,`  or a not closed single or double quote (`'`, `"`)

**Comment by dwertheimer** [comment]
Mike, thank you.Tthis plugin should not require a config, but should use
one of you have it. So I think you did find a bug. I will look at it later
today and let you know.

**Comment by mikeerickson** [comment]
@weyert @dwertheimer 

OK, I have discovered the issue.  Turns out, I *did* have an existing `templates/_configuration` file, but it was buried in another folder and I didn't realize it.  I now recall an issue I had sometime ago with the `/nn` plugin and it was determined at that time by @jgclark there was a bad default configuration file, but I never followed up to correct the issue.

So, this is the first time I have tried to use a plugin since that report, and I never corrected the `_configuration` file (or location).

I deleted the `templates/_configuration` file and recreated and all is fine now.
